package.json

```diff
@@ -95,9 +101,11 @@
     "file-loader": "1.1.11",
     "find-cache-dir": "1.0.0",
     "html-webpack-plugin": "2.30.1",
+    "ignore-styles": "5.0.1",
     "json-loader": "0.5.7",
     "json-server": "0.14.0",
     "local-web-server": "2.5.2",
+    "mocha": "5.2.0",
```

Comment:
Why not riteway/tape?

Reply:
I was having a little trouble getting riteway to work and thought that this also did a good job of answering the 5 questions. I would be happy to set up riteway if we can.

Reply:
Let's try to get riteway in place since the syntax is nicer and it has tape under the hood (speaking from the KISS philosophy here).

Reply:
I leave a good experience with Jest instead of Mocha. I changed the suite Mocha for Jest in an old project. For my surprise, when I change the suite, I could suppress

- mocha
- chai
- sinon
- proxyrequire
- istanbul

Jest can create mocks dependencies libraries super easy.

another good point and the best is this;

Jest parallelizes test runs across workers to maximize performance. Console messages are buffered and printed together with test results. Sandboxed test files and automatic global state resets for every test so no two tests conflict with each other.

Reply:
I don't think riteway has typescript types at the moment, it would need to be added to the library, that doesn't mean we can't use it, just that we would need to make a pr to the library before using it.

I thought we were going to continue to use alsatian for tests? are we looking into changing all our projects to another test library?

Reply:
+1 — is there a good reason not to continue using Alsatian?

Reply:
No reason. I was suggesting riteway/tape as an alternative to mocha/chai just based on better experience with it.

@warrenv Just pinging you FYI.

Reply:
@kennylavender Did you ever come up with a satisfactory answer/solution to the "5 questions" issue with alsatian?

Reply:
@warrenv in poet-js we found a format for Alsatian that answers the 5 questions. It's not a great format, it requires some duplication in the `@Test` decorator and method names. In my opinion, I think the way that the `@TestCase` decorator passes the values to the method creates some unnecessary complexity around 'how to reproduce', but it's not terrible.

Here is the file we created in poet-js

https://github.com/poetapp/poet-js/blob/master/test/Claim/IsValidClaim.ts

Reply:
Thanks. @jwicks31, can you have a look at the link @kennylavender referenced and see if it will apply here?

Reply:
@warrenv @kennylavender I set the example button test up in a similar way, but without using the `@TestCase` decorator. I may need a little direction if we would like to use the `@TestCase` decorator with React Components.

Reply:
We can jump on a call after the meeting if your still unsure or its not working.

Reply:
That would be cool just to make sure I am handling it in the right way.